src/Components/JobsContainer/JobsContainer.types.ts

```diff
@@ -17,6 +17,18 @@ export interface IJobsContainerProps {
 export interface IJobsContainerStyleProps {
     theme: IExtendedTheme;
 }
+
```

Comment:
we are probably going to want a types for that spans across all the components right? Can we make a `Jobs.types.ts` file for those common types that multiple components will use so we don't ahve other components referencing this component?